compiler/luci/import/src/GraphBuilderRegistry.cpp

```diff
@@ -161,6 +161,14 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   // BuiltinOperator_ARG_MAX = 56,
   // BuiltinOperator_HARD_SWISH = 117,
   // BuiltinOperator_DENSIFY = 124,
+
+#define CIRCLE_VNODE(OPCODE, CLASS) add(std::make_unique<CLASS>())
```

Comment:
Do we need `OPCODE`?

Reply:
No, we don't need, but I made this definition in accordance with definition for not virtual circle nodes (see above in this file).

Reply:
I think this is where lookup source is added.
We need a comment to express this is where tensor type(?) builders (CircleConst) are build.